src/api.v2/events/handleWorkRequest.js

```diff
@@ -21,10 +21,10 @@ const getSignedUrlIfAvailable = async (experimentId, ETag) => {
 
 
 const handleWorkRequest = async (Authorization, data) => {
-  const { experimentId } = data;
+  const { experimentId, body, requestProps } = data;
 
   // 1. Generate ETag for the new work request
-  const ETag = await generateETag(data);
+  const ETag = await generateETag({ experimentId, body, requestProps });
```

Comment:
This shoulnd't change anything, we are not using anything else from data in generateETag, this is just making this more clear